tasks.py

```diff
@@ -75,26 +75,40 @@ def task_wrapper(function=None):
 
 
 def docker_compose(context, command, **kwargs):
-    """Helper function for running a specific docker compose command with all appropriate parameters and environment.
+    """Helper function for running a specific docker-compose command with all appropriate parameters and environment.
 
     Args:
         context (obj): Used to run specific commands
-        command (str): Command string to append to the "docker compose ..." command, such as "build", "up", etc.
+        command (str): Command string to append to the "docker-compose ..." command, such as "build", "up", etc.
         **kwargs: Passed through to the context.run() call.
     """
     build_env = {
-        # Note: 'docker compose logs' will stop following after 60 seconds by default,
+        # Note: 'docker-compose logs' will stop following after 60 seconds by default,
         # so we are overriding that by setting this environment variable.
         "COMPOSE_HTTP_TIMEOUT": context.nautobot_floor_plan.compose_http_timeout,
         "NAUTOBOT_VER": context.nautobot_floor_plan.nautobot_ver,
         "PYTHON_VER": context.nautobot_floor_plan.python_ver,
     }
-    compose_command = f'docker compose --project-name {context.nautobot_floor_plan.project_name} --project-directory "{context.nautobot_floor_plan.compose_dir}"'
+    compose_command_tokens = [
+        "docker-compose",
```

Comment:
We should be using `docker compose` rather than `docker-compose` for new projects. Maybe an update needed in the cookiecutter?

Reply:
Revert this. The next version of the cookie has already been changed to this effect.

Reply:
Renamed `docker-compose` to `docker compose` in tasks.